ops/terraform/services/server/server-regression/main.tf

```diff
@@ -1,20 +1,19 @@
-locals {
-  default_tags = {
-    Environment    = local.seed_env
-    Layer          = local.layer
-    Name           = "bfd-${local.env}-${local.service}"
-    application    = "bfd"
-    business       = "oeda"
-    role           = local.service
-    stack          = local.env
-    Terraform      = true
-    tf_module_root = "ops/terraform/services/server/server-regression"
+module "terraservice" {
+  source = "../../_modules/bfd-terraservice"
+
+  environment_name     = terraform.workspace
+  relative_module_root = "ops/terraform/services/server/server-regression"
+  additional_tags = {
+    Layer = local.layer
+    Name  = "bfd-${local.env}-${local.service}"
+    role  = local.service
   }
+}
 
-  env              = terraform.workspace
-  established_envs = ["test", "prod-sbx", "prod"]
-  seed_env         = one([for x in local.established_envs : x if can(regex("${x}$$", local.env))])
-  is_ephemeral_env = !(contains(local.established_envs, local.env))
+locals {
+  default_tags = module.terraservice.default_tags
+  env          = module.terraservice.env
+  seed_env     = module.terraservice.seed_env
 
```

Comment:
Was the ommission of `is_ephemeral_environment` deliberate?

Reply:
That's intentional. This was mostly copy/pasta or _cargoculting_.